File: Year 3/Sem1/Mobile/crud_non_native/lib/Song.dart
class Song {
  String title;
  String artist;

  Song(this.title, this.artist);

  String getTitle() {
    return title;
  }

  void setTitle(String title) {
    this.title = title;
  }

  String getArtist() {
    return artist;
  }

  void setArtist(String artist) {
    this.artist = artist;
  }

  String toString() {
    return this.artist.toString() + " - " + this.title.toString();
  }


}

File: Year 3/Sem1/Mobile/CRUDNative/app/src/main/java/com/example/Domain/Song.java
package com.example.Domain;

import android.media.Image;

import java.io.Serializable;

public class Song implements Serializable {
    private String title;
    private String author;

    public Song(String title, String author) {
        this.title = title;
        this.author = author;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String toString() {
        return this.author.toString() + " - " + this.title.toString();
    }
}
